card/mod/bootstrap/set/all/bootstrap/table.rb

```diff
@@ -40,7 +40,7 @@ def table_cell cell
     if cell.is_a? Hash
       content_tag :td, cell.delete(:content).to_s.html_safe, cell
     else
-      content_tag :td, cell.html_safe
+      content_tag :td, String(cell).html_safe
```

Comment:
in general `Type()` is stronger than `to_type`. With `Type()` you are saying give me a `Type` if there is any sensible way of doing so (eg. `"a".to_a` fails but `Array("a")` returns `["a"]`) . 
But I double-checked for the String case and learned that `to_s` is so widely spread that there is no difference (in fact String() calls to_s on it's argument)